src/main/java/cuchaz/enigma/gui/elements/PopupMenuBar.java

```diff
@@ -24,79 +24,79 @@ public PopupMenuBar(Gui gui) {
 		{
 			JMenuItem menu = new JMenuItem(I18n.translate("popup_menu.rename"));
 			menu.addActionListener(event -> gui.startRename());
-			menu.setAccelerator(KeyStroke.getKeyStroke(KeyEvent.VK_R, 0));
+			menu.setAccelerator(KeyStroke.getKeyStroke(KeyEvent.VK_R, InputEvent.CTRL_MASK));
```

Comment:
```suggestion
			menu.setAccelerator(KeyStroke.getKeyStroke(KeyEvent.VK_R, InputEvent.CTRL_DOWN_MASK));
```
Along with similar changes below

Reply:
Done